edit_patient_registration_details.md

```diff
@@ -0,0 +1,23 @@
+###Editing  Patient Registration
+
+1.Login and on the home page click the Find Patient Link circled below
+
+![Register Patient Link](/assets/poc_find_patient_link.png)
+
+2.In the Search Bar, type any patient identifiers to search for the patient, the record of the patient appears as shown below.
+
+![Patient Search Record](/assets/poc_patient_record_search.png)
+
+3.Click on the last action that has Edit Demographics as shown below;
+
+![Edit Patient Demographics](/assets/poc_edit_patient_demographics.png)
+
+4.Edit Patient Information by clicking on the pencil icon next to a section header where those details belong
+
+![Registration Summary](/assets/poc_patient_summary.PNG)
+
+5.Confirm the details and Save the information.
+
+
```

Comment:
@solemabrothers add steps to edit the patient information from the clinican facing dashboard

Reply:
@ssmusoke we don't have the option of editing from the clinician facing dashboard, editing only happens on the registration summary. 
We will have to create a card to add this functionality